app/src/main/res/values/strings.xml

```diff
@@ -26,4 +26,25 @@
 
     <!-- Temp -->
     <string name="connecting">Connecting…</string>
+    <string name="create_meeting_event">Create Meeting event</string>
+    <string name="meeting_title_required">Meeting Title*</string>
+    <string name="enter_title">Enter Title</string>
+    <string name="start_date_required">Start Date*</string>
+    <string name="enter_date">Enter Date</string>
+    <string name="start_time_required">Start Time*</string>
+    <string name="enter_time">Enter Time</string>
+    <string name="end_date_optional">End Date</string>
+    <string name="location_optional">Location</string>
+    <string name="enter_location">Enter Location</string>
+    <string name="confirm">Confirm</string>
+    <string name="cancel">Cancel</string>
+    <string name="now">Now</string>
+    <string name="today">Today</string>
+    <string name="PAST_DATE_NOT_ALLOWED">Event has to start/end today or in the future</string>
+    <string name="END_DATE_AFTER_START_DATE_NOT_ALLOWED">End date has to be after or the same day as the start date</string>
+    <string name="START_DATE_AFTER_END_DATE_NOT_ALLOWED">Start date must be before or the same day as end date</string>
+    <string name="TIME_BEFORE_DATE_NOT_ALLOWED">Please, choose start and end dates before choosing time</string>
+    <string name="START_TIME_AFTER_END_TIME_NOT_ALLOWED">Start time must be before end time</string>
+    <string name="END_TIME_BEFORE_START_TIME_NOT_ALLOWED">End time must be after start time</string>
```

Comment:
Why are these names upper case?

Reply:
for no particular reason, I can change that